rm-docker-toolbox-setup.md

```diff
@@ -0,0 +1,28 @@
+# Docker Toolbox Setup
+
+This document outlines additional setup steps that need to be taken in order to set up the environment when using Docker Toolbox rather than Docker for Windows.
```

Comment:
Can you outline why you might want to use Docker Toolbox?

Reply:
Added in latest commit.